dadab/dadab.js

```diff
@@ -3,77 +3,101 @@ based on:
  - Simple Image Glitching Processing - datamoshing.com | 2016
  - a pixel sorting sketch from Jerome Martinez | 2017
  */
- 
+
 // output needs to be 1066 x 800
- 
+
+var capture;
+var gif;
 var img;
-var imgFileName = "vera";
-var fileType = "jpg";
+var imgFileName = 'vera';
```

Comment:
We'll need to change this, maybe to a hardcoded filename, and the file just gets overwritten every time a portrait is taken.